workflows/ena_utils/ena_api_requests.py

```diff
@@ -296,16 +315,38 @@ def create_ena_api_request(result_type, query, limit, fields, result_format="jso
     )
 
 
+class ENAAccessException(Exception): ...
+
+
+class ENAAvailabilityException(Exception): ...
+
+
 @task(
-    retries=2,
+    retries=RETRIES,
+    retry_delay_seconds=RETRY_DELAY,
     cache_key_fn=context_agnostic_task_input_hash,
     task_run_name="Get study from ENA: {accession}",
 )
-async def get_study_from_ena(accession: str, limit: int = 10) -> ena.models.Study:
+def get_study_from_ena(accession: str, limit: int = 10) -> ena.models.Study:
     logger = get_run_logger()
 
     logger.info(f"Will fetch from ENA Portal API Study {accession}")
 
+    is_public = is_ena_study_public(accession)
+    is_private = not is_public and is_ena_study_available_privately(accession)
```

Comment:
Mayber we should rename _is_private_ with _privately_available_?

Reply:
I would say that is slightly less precise, in that public studies are also _available_ privately (as in, with an authenticated request).